src/operations/create_claimable_balance.js

```diff
@@ -0,0 +1,63 @@
+import xdr from '../generated/stellar-xdr_generated';
+import { Asset } from '../asset';
+
+/**
+ * Create a new claimable balance operation.
+ * @function
+ * @alias Operation.createClaimableBalance
+ * @param {object} opts Options object
+ * @param {Asset} opts.asset - The asset for the claimable balance.
+ * @param {string} opts.amount - Amount.
+ * @param {Claimant[]} opts.claimants - An array of Claimants
+ * @param {string} [opts.source] - The source account for the operation. Defaults to the transaction's source account.
+ * @returns {xdr.Operation} Create claimable balance operation
+ *
+ * @example
+ * const asset = new Asset(
+ *   'USD',
+ *   'GDGU5OAPHNPU5UCLE5RDJHG7PXZFQYWKCFOEXSXNMR6KRQRI5T6XXCD7'
+ * );
+ * const amount = '100.0000000';
+ * const claimants = [
+ *   new Claimant(
+ *     'GCEZWKCA5VLDNRLN3RPRJMRZOX3Z6G5CHCGSNFHEYVXM3XOJMDS674JZ',
+ *      Claimant.predicateBeforeAbsoluteTime("4102444800000")
+ *   )
+ * ];
+ *
+ * const op = Operation.createClaimableBalance({
+ *   asset,
+ *   amount,
+ *   claimants
+ * });
+ *
+ */
+export function createClaimableBalance(opts) {
+  if (!(opts.asset instanceof Asset)) {
+    throw new Error(
+      'must provide an asset for create claimable balance operation'
+    );
+  }
+
+  if (!this.isValidAmount(opts.amount)) {
+    throw new TypeError(this.constructAmountRequirementsError('amount'));
+  }
+
+  if (!opts.claimants || (opts.claimants && opts.claimants.length === 0)) {
```

Comment:
Doesn't the latter condition fulfill the former condition? 
That is, if `opts.claimants = []`, then I think `!opts.claimants` will be true.
Maybe a type check for `Array` makes sense here, instead?

Reply:
good point,  updated to. `(!Array.isArray(opts.claimants) || opts.claimants.length === 0)`